views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/recorder.js

```diff
@@ -294,9 +311,30 @@ define([
                 this._interruptRecording();
 
                 provider.cancel();
+                setState(recorder, recorderStates.CANCEL);
+
+                this.releaseResourses(); // can release resources because provider don't send data
+
                 this.trigger('cancel');
             },
 
+            /**
+             * Release resourses after stop recording
+             */
+            releaseResourses: function releaseResourses() {
+                if (mediaStream) {
+                    _.forEach(mediaStream.getTracks(), function(track) {
```

Comment:
Caution, `getTracks()` is not compatible with IE: https://developer.mozilla.org/en-US/docs/Web/API/MediaStream/getTracks

Could you at least check if the method exists?

Reply:
Edge, IE and Safari don't support the mediaRecorder API as specified in this document:
https://docs.google.com/document/d/1H9qb1L8edYmKyglSc5VnYAVKVkj3eKFf8Gj1ejVrOHA/edit